diff --git a/Makefile b/Makefile
index 1ac195c..aae9680 100644
--- a/Makefile
+++ b/Makefile
@@ -3,8 +3,8 @@ all: input-event-daemon docs/input-event-daemon.8 docs/input-event-daemon.html
 input-event-daemon: input-event-daemon.c input-event-daemon.h input-event-table.h
 	$(CC) $(CFLAGS) $< $(LDFLAGS) -o $@
 
-input-event-table.h: /usr/include/linux/input.h
-	awk -f parse_input_h.awk < $< > $@
+#input-event-table.h: /usr/include/linux/input.h
+#	awk -f parse_input_h.awk < $< > $@
 
 docs/input-event-daemon.8: docs/input-event-daemon.txt
 	a2x -f manpage $<
